docs/setup.md

```diff
@@ -10,10 +10,10 @@
 
 ## Prerequisites
 
-|Solution Pack Name|Purpose|
-| :- | :- |
-|SOAR Framework 1.0.0|Required for Incident Response modules|
-|SOC Simulator 1.0.1|Required for Scenario Module and SOC Simulator connector|
+| Solution Pack Name | Version | Purpose                                                  |
+|:-------------------|:-------:|:---------------------------------------------------------|
+| SOAR Framework     | v1.0.0  | Required for Incident Response modules                   |
+| SOC Simulator      | v1.0.1  | Required for Scenario Module and SOC Simulator connector |
```

Comment:
SOC simulator version >> 1.0.2
SFSP >> 2.1.0
@cs-rohit-khune @cs-abhishek-shukla : Please confirm.

Reply:
The SP will work fine on lower versions of SFSP and SOC SP.  We haven't made any version-dependent changes.